package.json

```diff
@@ -18,57 +18,61 @@
   "license": "MIT",
   "jest": {
     "setupFiles": [
-      "<rootDir>/shim.js"
+      "<rootDir>/config/jest/shim.js"
     ],
     "mapCoverage": true,
     "collectCoverageFrom": [
       "app/**/*.{js,jsx,ts,tsx}"
     ],
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/config/jest/assetsTransformer.js",
+      "\\.(css|less)$": "<rootDir>/config/jest/assetsTransformer.js"
+    },
     "testMatch": [
       "**/?(*.)(spec|test).js?(x)"
     ]
   },
   "dependencies": {
     "cross-env": "5.1.1",
-    "react": "16.1.1",
-    "react-dom": "16.1.1",
-    "react-hot-loader": "3.1.2",
-    "react-router-dom": "4.2.0"
+    "react": "16.2.0",
+    "react-dom": "16.2.0",
+    "react-hot-loader": "3.1.3",
+    "react-router-dom": "4.2.2"
   },
   "devDependencies": {
     "babel-cli": "6.26.0",
     "babel-core": "6.26.0",
-    "babel-eslint": "7.2.3",
+    "babel-eslint": "8.0.2",
     "babel-loader": "7.1.2",
     "babel-preset-es2015": "6.24.1",
     "babel-preset-react": "6.24.1",
     "babel-preset-react-boilerplate": "1.1.1",
     "babel-preset-react-hmre": "1.1.1",
     "babel-preset-stage-2": "6.24.1",
-    "copy-webpack-plugin": "4.1.1",
+    "copy-webpack-plugin": "4.2.3",
     "css-loader": "0.28.7",
     "enzyme": "3.2.0",
     "enzyme-adapter-react-16": "1.1.0",
-    "eslint": "3.19.0",
-    "eslint-config-airbnb": "15.1.0",
-    "eslint-loader": "1.8.0",
+    "eslint": "4.12.1",
```

Comment:
does `4.21.1` work with the `eslint-loader`? Have you tested it

Reply:
Yes! I have tested everything (with webpack dev & prod) and have pushed the code. You can pull the branch and check it once if you like.

Reply:
Nice!